app/pb_kits/playbook/pb_radio/_radio.html.erb

```diff
@@ -0,0 +1,17 @@
+<%= content_tag(:label,
+    id: object.id,
+    data: object.data,
+    text: object.text,
+    value: object.value,
+    checked: object.checked,
+    class: object.classname) do %>
+
+
+    <input type="radio" value="<%= object.value %>" name="<%= object.name %>" <%= object.selected %>>
+    <span class="pb_radio_button">
+      <div class="radio"></div>
+    </span>
```

Comment:
I think this is unnecessary and done through CSS...that said I could be convinced otherwise. Thoughts @thestephenmarshall & @markiearnold